### PR TITLE
added cart page

### PR DESCRIPTION
**Pull request by shivanigangadharan**
image

### CODE REVIEW

File: pages/cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    display: flex;
+    gap: 20px;
+    align-items: center;
+}
+.qty-ip{
+    width: 2rem;
+    border: none;
+    outline: none;
+    text-align: center;
+    padding: 4px 8px;
```
Comment (spraveenofficial) [inline pages/cart/cart.css]: Try to follow one pattern, either rem or px.

File: pages/cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
+.cart-page{
+    background: var(--lightergrey);
+    padding: 8rem 0px;
+    height: 100vh;
+}
+.cart-content{
+    display: flex;
+    gap: 20px;
```
Comment (spraveenofficial) [inline pages/cart/cart.css]: Here too.

**Review by spraveenofficial — COMMENTED** [review]
Suggested few changes

File: pages/cart/cart.css (hunk 2 above)
Comment (shivanigangadharan) [reply pages/cart/cart.css]: Done [180bddf](https://github.com/shivanigangadharan/e-mart/pull/5/commits/180bddf41e1c821776aaeabf6578dabc9fc2b09e)

File: pages/cart/cart.css (hunk 1 above)
Comment (shivanigangadharan) [reply pages/cart/cart.css]: Done [180bddf](https://github.com/shivanigangadharan/e-mart/pull/5/commits/180bddf41e1c821776aaeabf6578dabc9fc2b09e)

File: pages/cart/cart.html [hunk 3]
```diff
@@ -37,7 +37,59 @@
...
+                <div class="flex-between">
+                    <span>Delivery charges </span>
+                    <span class="text-right"> 499 </span>
+                </div>
+                <hr>
+                <div class="flex-between">
+                    <span class="bold">TOTAL AMOUNT</span>
+                    <span class="text-right bold">3499</span>
+                </div>
+                You will save Rs. 1999 on this order
```
Comment (bhanuprakashj) [inline pages/cart/cart.html]: wrap this text into p/span element, and hr is not flexible for styling rather using border property is suggested here

**Review by spraveenofficial — APPROVED** [review]
Good to merge now.

File: pages/cart/cart.css [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+.cart-content{
+    display: flex;
+    gap: 1.25rem;
+    justify-content: center;
+}
+.flex-between{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 1rem 0px;
```
Comment (Prabhav-Sharma) [inline pages/cart/cart.css:15]: Can use global variables for padding, gap and margins

File: pages/cart/cart.html (hunk 3 above)
Comment (shivanigangadharan) [reply pages/cart/cart.html]: Done in [305ee5e](https://github.com/shivanigangadharan/e-mart/pull/5/commits/305ee5e42c1605deb09a9b070bba7c228ecfbdeb), and styled hr using border property.

File: pages/cart/cart.css (hunk 4 above)
Comment (shivanigangadharan) [reply pages/cart/cart.css:15]: Sure, will do this in the other branch in entire proj

**Pull request merged**